aiohttp/web_ws.py

```diff
@@ -323,6 +323,16 @@ def ws_protocol(self) -> Optional[str]:
     def compress(self) -> bool:
         return self._compress
 
+    def get_extra_info(self, name: str, default: Any = None) -> Any:
```

Comment:
Can the ambiguous "extra info" be surfaced as something more specific? It seems weird to me, putting this up as a public API.

Reply:
Good point, but I decided to be consistent with already existing https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientWebSocketResponse.get_extra_info

Reply:
Fair enough. Marking this thread as resolved.